ui-guide/Components/Icons.purs

```diff
@@ -71,6 +71,7 @@ component =
             , Tuple "Tiktok" $ Icon.tiktok [ css "text-tiktok-green" ]
             , Tuple "Twitter" $ Icon.twitter [ css "text-twitter-blue" ]
             , Tuple "Youtube" $ Icon.youtube [ css "text-youtube-red" ]
+            , Tuple "Outbrain" $ Icon.outbrain_
```

Comment:
Can we move this up between `LinkedIn` and `Pinterest`?

Also, it looks like outbrain's color is this orange: `#ee6512`

Can we add `'outbrain-orange': '#ee6512'` here: https://github.com/citizennet/purescript-ocelot/blob/22bc010195b76a3dbd708e49adbfd0fd869a6b9b/css/src/tailwind.js#L75-L76

Reply:
Sure I'll update

Reply:
Added

Reply:
Thanks! The icon is still out of alphabetical order though, can we bump it up between LinkedIn and Pinterest?

Reply:
Sure